### ISSUE TITLE
alpine-node docker image and google-cloud equals error loading ld-linux-x86-64.so.2

### ISSUE THREAD

**Issue by Rich-amoebaWare** [GoogleCloudPlatform/google-cloud-node#1754]
- OS:
Linux 4.4.20-moby (a docker image from mhart/alpine-node:6.9.1
  - Node.js version:
6.9.1 (and back to at least 6.7.0)
  - npm version:
3.10.8
  - google-cloud-node version:
0.43.0

I started using the google-cloud sdk a few weeks ago and was using the mhart/alpine-node image for its "slenderness". When I introduced google-cloud into my code I went to build my docker file using alpine-node to discover the following error:

module.js:597
  return process.dlopen(module, path._makeLong(filename));
                 ^

Error: Error loading shared library ld-linux-x86-64.so.2: No such file or directory (needed by /usr/src/node_modules/google-cloud/node_modules/grpc/src/node/extension_binary/grpc_node.node)
    at Error (native)
    at Object.Module._extensions..node (module.js:597:18)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/src/node_modules/google-cloud/node_modules/grpc/src/node/src/grpc_extension.js:38:15)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)

npm ERR! Linux 4.4.20-moby
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "run" "test-env"
npm ERR! node v6.9.1
npm ERR! npm  v3.10.8
npm ERR! code ELIFECYCLE
npm ERR! enabled@0.4.0 test-env: `export NODE_ENV=test&&node index.js`
npm ERR! Exit status 1

I switched to using the normal node docker image based on debian and of course I don't get the error and everything works well.

However, I would like to switch back to alpine-node to get rid of all the weight...

How can I fix this error? 

Thanks

**Comment by stephenplusplus** [comment]
This issue was moved to grpc/grpc#8528

**Comment by stephenplusplus** [comment]
Thanks for reporting. I moved the issue to the grpc repo. I'm subscribed on that thread, so if something needs to be done here, we will be on it.

**Comment by Rich-amoebaWare** [comment]
thanks. I don't think its an issue with your code... I had to put c programmer hat on and remember that alpine linux uses musl and the reference to ld-linux-x86--64.so.2 was likely built with glibc. So I installed the lib6compat package while building my alpine image and the error (which occurred on start up) went away.... which may have solved my issue...but I have another issue once that occurs after that one went away...that I have to get beyond before I know for sure.
